### PR TITLE
include 360 on rinkeby reporting

### PR DESCRIPTION
**Pull request by akremstudy**
### Summary
- refactored cli/commands/report.py to enable 360 Flex reporting on rinkeby.
- also edited the reporter_utils.py file to handle function signature differences in 360 and Tellorx Oracles

### Steps Taken to QA Changes
- staked and reported on rinkeby. Txn hash: 0x38bb5872a070870cb805086b695dd0831ca7ea75cb6c0609f99fb00d802c346f. Has to be checked manually since rinkeby etherscan has been deprecated.

- further testing later

### Checklist
This pull request is:

- [ ] A documentation error, docs update, or typographical error fix
	- No tests or issue needed
- [ ] A code fix
	- Please reference the related issue by including "Closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a bug report issue 
	- Please include tests. Fixes without tests will not be accepted unless it's related to the documentation only.
    - Please make sure docs are updated if need be
- [x] A feature implementation
	- Please reference the related issue by including "Closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a feature enhancement issue 
	- Please include tests
    - Please make sure docs updates are both thorough and easy to reproduce by somebody with limited knowledge of the feature that you are submitting

**Happy engineering!**

### CODE REVIEW

**Review by oraclown — CHANGES_REQUESTED** [review]
Thanks @akremstudy . I added some questions/suggestions

File: src/telliot_feeds/cli/commands/report.py [hunk 1]
```diff
@@ -548,6 +472,62 @@ async def report(
...
+
+            if oracle_address:
+                contracts.oracle.address = oracle_address
+                contracts.oracle.connect()
+
+            if autopay_address:
+                contracts.autopay.address = autopay_address
+                contracts.autopay.connect()
+
+            # Type 2 transactions unsupported currently
```
Comment (oraclown) [inline src/telliot_feeds/cli/commands/report.py:488]: Why?

**Review by oraclown — APPROVED** [review]
Awesome @akremstudy , ty!

Are reporting tests passing for diva/rng updated reporters?

File: src/telliot_feeds/cli/commands/report.py (hunk 1 above)
Comment (akremstudy) [reply src/telliot_feeds/cli/commands/report.py:488]: idk that was from something previous

**Comment by akremstudy** [comment]
> Awesome @akremstudy , ty!
> 
> Are reporting tests passing for diva/rng updated reporters?

yes they are

**Pull request merged**